tic_tac_toe/game.py

```diff
@@ -9,7 +11,12 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
+    query_row = position[0]
```

Comment:
Remember you can do `row, col = position`, which holds true for all collections as long as the number of variables is equal to the number of elements.